bleak/backends/bluezdbus/manager.py

```diff
@@ -471,10 +471,47 @@ async def passive_scan(
 
                 assert_reply(reply)
 
-                # It is important to export after registering, otherwise BlueZ
-                # won't use the monitor
+                # It is important to export AFTER registering, otherwise BlueZ won't use the monitor
                 self._bus.export(monitor_path, monitor)
 
+                # During export, MGMT command "Add Advertisement Patterns Monitor (0x0052)" is executed.
```

Comment:
Would it work to detect `"org.bluez.AdvertisementMonitorManager1"` in the adapter details for this case?

https://github.com/Bluetooth-Devices/bluetooth-adapters/blob/bcadadd67150bd4648a0002dc151adc46d302732/src/bluetooth_adapters/systems/linux.py#L104

Reply:
Thank you for the suggestion, but unfortunately this would not solve this particular issue.

 The issue here is that not only that `org.bluez.AdvertisementMonitorManager1` is present in managed objects, even calling `org.bluez.AdvertisementMonitorManager1.RegisterMonitor()` succeeds. Only "later" kernel responds with `Unknown Command (0x01)` to underlying MGMT command `Add Advertisement Patterns Monitor (0x0052)` - this response is processed by the BlueZ code and registered `AdvertisementMonitor1` is destroyed, but the error is not propagated to the D-Bus caller (bleak) in any way.